### PR TITLE
feat: add private locations landingpage

### PR DESCRIPTION
**Pull request by schoenwaldnils**
Added a new landing page on https://checklyhq-com-git-feat-private-locations-checkly.vercel.app/private-locations/

WIP: 
- [x] set links on buttons
- [x] edit page description
- [x] edit alt texts on images
(here `site/content/private-locations/_index.md`)

### CODE REVIEW

File: site/layouts/private-locations/list.html [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+              updateDom()
+            }
+
+            let interval = setInterval(moveToNextVideo, 1000 * 24) // 24s
+
+            const reset = (next) => {
+              clearInterval(interval)
+
+              currentvideo = next
+              console.log(currentvideo)
```
Comment (stefanjudis) [inline site/layouts/private-locations/list.html:152]: @schoenwaldnils `console.log` spotted. :)

Comment (schoenwaldnils) [reply site/layouts/private-locations/list.html]: Good catch 😁

**Comment by orcunilbeyli** [comment]
## Hero
- Width of text and the terminal should match.
SCR-20220620-lim
- Terminal shouldn't be image. It should be selectable text and the cursor in the end should blink forever. As is in real terminal. Please double check documentation 'layer group' in figma.
image

## How it works?
- Videos should have border radius and solid border color. Please check design.
SCR-20220620-lx4
- When user clicks on the text, videos should restart from 00:00.
- I should see next video on every pne page scroll. Let's discuss this @schoenwaldnils  

## Get started
- "Create private location" vertical alignment is off. It should optically aligned, not mathematically.
SCR-20220620-lky
- "Install the container" should have selectable text
SCR-20220620-luf
- "Create checks" vertical alignment is off. It should optically aligned, not mathematically.
SCR-20220620-lly

**Comment by orcunilbeyli** [comment]
### Implementation
image

### Design
image

Hero
- It should say "Private Locations" instead "Terraform"
- Navigation and hero section's background color should be same. Currently it is white and gray. (Like our [home page](https://www.checklyhq.com/).)
- Spacing between navigation and hero section should be smaller. 
image

- `h1` tag
   - color, font-size, line-height, letter-spacing is incorrect.
- `p`
   - font-size is incorrect
- Get started button padding is incorrect.
- There shouldn't be horizontal scrollbar in the terminal.
image

**Pull request merged**